pkgs/servers/matrix-synapse/default.nix

```diff
@@ -53,6 +53,7 @@ buildPythonApplication rec {
     typing-extensions
     authlib
     pyjwt
+    hiredis
```

Comment:
Redis integration is only used when the process is split up into multiple workers. Is that something you plan to do?

Reply:
Without adding this dependency, the build would've failed locally. OTOH I don't see a change in the relevant files, so I may need to recheck that.

Reply:
So you're right, this is strictly speaking an optional dependency. Should we exclude it for now or leave it for folks who want to do some work to get workers running?

Reply:
I don't mind this at all.

Reply:
The best would maybe be to add a `enableRedis ? false` flag to the derivation if it doesn't work with the nixos module, and to add this as `lib.optional`?

Reply:
Fixed %)